False positive when multiplying against an untyped int constant, again

I picked up this linter in the latest golangci-lint release (includes 0.0.3) and am trying with the latest 0.0.4 version as well. I am still seeing the previously-resolved false positive. The only difference I can think of might be because my constant is in a different package?

package session:

```
const TokenExp = 10 * 60
```

package service (test file):

```
require.WithinDuration(t, time.Now().Add(session.TokenExp*time.Second), loginResp.ExpiresAt.AsTime(),
2*time.Second)
```

Result:

```
$ durationcheck ./...
.../session_test.go:54:4: Multiplication of durations: `session.TokenExp * time.Second`
```

--- a/durationcheck.go
+++ b/durationcheck.go
@@ -147,8 +147,7 @@ func isAcceptableNestedExpr(pass *analysis.Pass, n ast.Expr) bool {
 		t := pass.TypesInfo.TypeOf(e)
 		return !isDuration(t)
 	case *ast.SelectorExpr:
-		t := pass.TypesInfo.TypeOf(e)
-		return !isDuration(t)
+		return isAcceptableNestedExpr(pass, e.X) && isAcceptableIdent(pass, e.Sel)
 	}
 
 	return false

--- a/testdata/src/a/a.go
+++ b/testdata/src/a/a.go
@@ -1,6 +1,7 @@
 package a
 
 import (
+	"b"
 	"time"
 )
 
@@ -53,6 +54,10 @@ func validCases() {
 	_ = time.Duration(ms.fieldA) * time.Second
 
 	_ = time.Second * time.Duration(ms.fieldA)
+
+	_ = b.SomeInt * time.Second
+
+	_ = time.Second * b.SomeInt
 }
 
 func invalidCases() {
@@ -80,6 +85,10 @@ func invalidCases() {
 	_ = ms.fieldB * time.Second // want `Multiplication of durations`
 
 	_ = time.Second * ms.fieldB // want `Multiplication of durations`
+
+	_ = b.SomeDuration * time.Second // want `Multiplication of durations`
+
+	_ = time.Second * b.SomeDuration // want `Multiplication of durations`
 }
 
 func someDuration() time.Duration {

--- a/testdata/src/b/b.go
+++ b/testdata/src/b/b.go
@@ -1,0 +1,8 @@
+package b
+
+import "time"
+
+const (
+	SomeInt      = 10
+	SomeDuration = 1 * time.Second
+)

